src/main/interface/settings.c

```diff
@@ -1112,6 +1114,10 @@ const clivalue_t valueTable[] = {
     { "frsky_spi_bind_hop_data",        VAR_UINT8   | MASTER_VALUE | MODE_ARRAY, .config.array.length = 50, PG_RX_FRSKY_SPI_CONFIG, offsetof(rxFrSkySpiConfig_t, bindHopData) },
     { "frsky_x_rx_num",                 VAR_UINT8   | MASTER_VALUE, .config.minmax = { 0, 255 }, PG_RX_FRSKY_SPI_CONFIG, offsetof(rxFrSkySpiConfig_t, rxNum) },
     { "frsky_spi_use_external_adc",     VAR_UINT8   | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_RX_FRSKY_SPI_CONFIG, offsetof(rxFrSkySpiConfig_t, useExternalAdc) },
+#endif
+#ifdef USE_RX_SFHSS_SPI
+    { "sfhss_tx_id",                    VAR_UINT8   | MASTER_VALUE | MODE_ARRAY, .config.array.length = 2, PG_RX_SFHSS_SPI_CONFIG, offsetof(rxSfhssSpiConfig_t, bindTxId) },
```

Comment:
What is the reason for not using `rsky_spi_tx_id`, `frsky_spi_offset` for this, and does the user get any value out of this?

Reply:
Change SFHSS bind parameter rxSfhssSpiConfig to rxFrSkySpiConfig.
Remove rxSfhssSpiConfig.